environment.yml

```diff
@@ -0,0 +1,30 @@
+name: mgstellargraph
+
+channels:
+- conda-forge
+- stellargraph
+
+dependencies:
+# dev environment
+  - python=3.7
+  - coverage
+  - pytest
+  - pytest-cov
+  - black
+  - conda-forge::pre-commit
+  - setuptools
+
+# metagraph dependencies
```

Comment:
I don't see `metagraph` listed here. Rather than duplicating the metagraph dependencies, you should simply list `metagraph::metagraph` (it's on the metagraph channel currently) as a dependency and many of the others become automatic dependencies as a result.

Reply:
https://github.com/metagraph-dev/metagraph-stellargraph/pull/1/commits/da906041d7d47232ee0ea75327e8c3f2c435e498 implements this suggestion.